### PR TITLE
update firehose version to 0.2.0

### PR DESCRIPTION
**Pull request by 8naama**
Seems like these pages were missed when the version was last updated

### CODE REVIEW

**Review by Simplychee — APPROVED** [review]
Looks good to me 💃

**Comment by 8naama** [comment]
@Simplychee @nico-shishkin Sorry, we will update the version today (hopfully) so I updated this PR to it;
Also added a small new explanation on a new field

File: docs/shipping/AWS/aws-api-gateway.md [hunk 1]
```diff
@@ -58,7 +58,8 @@ Specify the stack details as per the table below, check the checkboxes and selec
...
 | `services`                                 | A comma-seperated list of services you want to collect logs from. Supported options are: `apigateway`, `rds`, `cloudhsm`, `cloudtrail`, `codebuild`, `connect`, `elasticbeanstalk`, `ecs`, `eks`, `aws-glue`, `aws-iot`, `lambda`, `macie`, `amazon-mq`. | -                 |
-| `customLogGroups`                          | A comma-seperated list of custom log groups you want to collect logs from                                                                                                                                                                                | -                 |
+| `customLogGroups`                          | A comma-seperated list of custom log groups you want to collect logs, or the ARN of the Secret parameter (explanation below) storing the log groups list if it exceeds 4096 characters. from                                                                                                                                                                                | -                 |
```
Comment (Simplychee) [inline docs/shipping/AWS/aws-api-gateway.md:61]: @8naama Secret parameter = useCustomLogGroupsFromSecret ?

**Review by Simplychee — APPROVED** [review]

File: docs/shipping/AWS/aws-api-gateway.md (hunk 1 above)
Comment (8naama) [reply docs/shipping/AWS/aws-api-gateway.md:61]: @Simplychee No, `useCustomLogGroupsFromSecret` is a parameter in the stack which indicates if they used a secret or not (`true` or `false`), if it's `true` >> then they need to place their secret ARN in `customLogGroups`
It's the explanation here:
https://github.com/logzio/documentation/pull/565/files#diff-3128cc9d1bdf5001fe5ebbcf6db9a67d65d568377fdb5f1ad07cee170b907a44R74-R86

I'll add link to it, maybe will be more clear?

**Review by Simplychee — APPROVED** [review]

**Pull request merged**